ui/packages/consul-ui/translations/routes/en-us.yaml

```diff
@@ -35,17 +37,17 @@ dc:
         body: |
           <ul>
             <li>
-              <a href="{CONSUL_DOCS_URL}/enterprise/license/faq#q-is-there-a-grace-period-when-licenses-expire">
+              <a href="{CONSUL_DOCS_URL}/enterprise/license/faq#q-is-there-a-grace-period-when-licenses-expire" target="_blank" rel="noopener noreferrer">
                 License expiration
               </a>
             </li>
             <li>
-              <a href="{CONSUL_DOCS_URL}/enterprise/license/faq#q-how-can-i-renew-a-license" target="_blank" rel="noopener noreferrer">
+              <a href="{CONSUL_DOCS_URL}/docs/enterprise/license/faq#q-how-can-i-renew-a-license" target="_blank" rel="noopener noreferrer">
```

Comment:
Didn't I see another PR go in somewhere to fix this up (I don't think the `docs` bit should be there). I'm not totally sure how this will merge down, so maybe its worth merging this anyway and then following up with a quick once over and correction once all these backports are in. I'll leave it up to you how you want to do this but we need to finish up without this extra `docs` in there

Reply:
This might be the result of a bad merge conflict resolution from me. The original PR (#12792) has the changes that were submitted to make this change for the release. I'll double check this to ensure it reflects the intended change.

Reply:
This was the PR to fix the `docs/docs` broken link here: https://github.com/hashicorp/consul/pull/12815